### ISSUE TITLE
Spotify API Streaming Endpoint - Queue

### ISSUE THREAD

**Issue by supreme99** [spotify/web-api#1388]
Hi dear Spotify Team. 

First of all, I want to appreciate your work on the API. It works very well. There are many useful Endpoint to create some nice Applications. But there's one feature missing, which I would like to have. As I know its not possible to edit, add or delete the users queue. I think this would be a nice feature for the `/v1/me/player` endpoint. Do you already have this in work or has this feature already been denied?

I would like to be able to add, rerange, delete and see what's in the current users queue. Thank you for your work. 

kind regards

Yanik

**Comment by turang** [comment]
However it's duplicate of #462; i'm still waiting for this feature too. :(

**Comment by supreme99** [comment]
@turang Yes I know. But your issue is a few years ago and I tought a new issue could be usefull to get their attention again
